samples/bluetooth/peripheral_uart_nrf53/cpuapp/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+cmake_minimum_required(VERSION 3.8)
+
+set(NRF_SUPPORTED_BOARDS
+  nrf5340_dk_nrf5340_cpuapp
+  nrf5340_dk_nrf5340_cpuappns
+)
+
+include(../../../../cmake/boilerplate.cmake)
```

Comment:
This needs to be:

```
include($ENV{ZEPHYR_BASE}/../nrf/cmake/boilerplate.cmake)
```

matching other examples or this sample won't build at all if moved to a different directory.